Scene3.js

```diff
@@ -0,0 +1,22 @@
+class Scene3 extends Phaser.Scene {
+	constructor() {
+		
+		super("StartGame");
+	}
+    preload(){
+      
+    }
+    create(){
+      
+        
+        // onclick on text switch to scene2
+        const clickButton = this.add.text(80, 100, 'Start Game', { fill: '#0f0' })
```

Comment:
Style the Start Game as a button using Sprites like - 
![start_button](https://user-images.githubusercontent.com/46218536/137613356-e900e210-f533-4412-883c-1b2b853ffc29.png)


Also, use the same background image on Start Game Scene as well (It should not be plain black).

When the user hovers on the Start Game button, the Mouse pointer should convert into clickable icon,